js/core/tty/terminal.js

```diff
@@ -35,6 +35,9 @@ refresh();
 exports.color = vga.color;
 
 exports.print = function(text, repeat, fg, bg) {
+  // fix issue #53 where non-strings (ints, etc...) would not get printed
+  text = String(text);
```

Comment:
@piranna `text = text.toString()` will fail if text is null/undefined. `String(expression)` converts any value to string, while `new String(expression)` creates an object